iocage/lib/Config/Jail/Properties/Defaultrouter.py

```diff
@@ -126,7 +129,12 @@ def __init__(
     def apply(self, jail: 'iocage.lib.Jail.JailGenerator') -> typing.List[str]:
         """Return a list of commands that configure the default IPv4 route."""
         commands: typing.List[str] = []
-        gateway = self._gateway_address
+        gateway_address = self._gateway_address
+
+        if gateway_address is None:
+            return []
```

Comment:
how come mypy doesn't complain about us returning `[]` here, when our return type is `List[str]`?